recipe/build.sh

```diff
@@ -5,7 +5,8 @@ export CFLAGS="-DACCEPT_USE_OF_DEPRECATED_PROJ_API_H=1  ${CFLAGS}"
 
 if [[ $(uname) == 'Darwin' ]]; then
   export CXXFLAGS="-fPIC -DPIC -g -O2 -std=c++11 -stdlib=libc++ ${CFLAGS}"
-  ARGS="--disable-openmp"
+  export LDFLAGS="${LDFLAGS} -fopenmp"
```

Comment:
You don't need this hack in this package. In the below command line, you'll see that `-fopenmp` is added by this hack and again by the build system
```
libtool: link: x86_64-apple-darwin13.4.0-clang -DACCEPT_USE_OF_DEPRECATED_PROJ_API_H=1 -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -I$PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/cdo-1.9.7.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -D_THREAD_SAFE -pthread -Wl,-pie -Wl,-headerpad_max_install_names -Wl,-dead_strip_dylibs -Wl,-rpath -Wl,$PREFIX/lib -fopenmp -o cdi cdi.o printinfo.o  -L$PREFIX/lib ../src/.libs/libcdi.a -leccodes -lhdf5 -lnetcdf -lm $PREFIX/lib/libuuid.dylib -fopenmp -pthread
```

Reply:
Cool. I'l remove it in the next iteration of this package.

Reply:
yh, CDO's build-system adds openmp flags if the compiler supports it.